dfg

```diff
@@ -0,0 +1,409 @@
+[33mcommit 772c302c9eb294bdfe1733541f8121bdf58b7319[m
```

Comment:
Did you intend to commit this file as well?

Reply:
ha, this is commit history IIRC, which shouldn't be included.

Reply:
uncommit this from pull request?

Reply:
Could you help figuring out how to uncommit this later today?

Reply:
My git is rusty nowadays.

But google gives me this post: https://devconnected.com/how-to-remove-files-from-git-commit/